docs/capabilities/app-client.md

```diff
@@ -108,9 +108,9 @@ The create method is a wrapper over the `appCreate` (bare calls) and `appCreateM
 
 ```typescript
 // Use no-argument bare-call
-const { result, app } = factory.create()
+const { result, appClient } = factory.send.bare.create()
 // Specify parameters for bare-call and override other parameters
-const { result, app } = factory.create({
+const { result, appClient } = factory.send.bare.create({
```

Comment:
We also settled on `appClient` over `app` to meet your principle of clear, explicit naming @SilentRhetoric